csharp/test/Microsoft.ML.OnnxRuntime.Tests/Microsoft.ML.OnnxRuntime.Tests.csproj

```diff
@@ -72,30 +72,31 @@
     <ProjectReference Include="$(OnnxRuntimeCsharpRoot)\src\Microsoft.ML.OnnxRuntime\Microsoft.ML.OnnxRuntime.csproj" />
   </ItemGroup>
 
-  <Target Name="ProtoGen" BeforeTargets="Build">
+  <!-- generate OnnxMl.cs from ONNX protobuf definition -->
+  <Target Name="ProtoGen" BeforeTargets="BeforeBuild">
     <Exec Command="$(ProtocExe) -I=$(ProtoSrc) --csharp_out=. $(ProtoSrc)\onnx-ml.proto3" ContinueOnError="false"></Exec>
```

Comment:
'Build' target may be empty and not fire. 'BeforeBuild' works more reliably in my testing.